pkg/notifications/smtp/smtp_test.go

```diff
@@ -1,214 +1,218 @@
 package smtp
```

Comment:
we still need to fix this test, but it's not crucial at all, I will open a dedicate issue for that :) 
just adding a comment to remember about this